### PR TITLE
Tests refactoring: make tests more structured.

### PR DESCRIPTION
**Pull request by sergiitk**
Refactor tests:
- Introduce common helpers file
- Add `test/bootstrap.test.js` to make initial setup and run tests in order
- Split `PhoenixClient` constructor tests from `User` tests

### CODE REVIEW

File: test/helpers/test-helper.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+/**
+ * Imports.
+ */
+const PhoenixClient = require('../../lib/phoenix-client');
```
Comment (deezone) [inline test/helpers/test-helper.js:6]: Any way of referencing the root rather than going up and down the path?

**Comment by deezone** [comment]
Just the one comment, otherwise 👍

File: test/helpers/test-helper.js (hunk 1 above)
Comment (sergiitk) [inline test/helpers/test-helper.js:6]: As I understand, this is a common practice in Node.js.
Though, I don't like it either. @DFurnes is there a better way to do it?

Comment (DFurnes) [inline test/helpers/test-helper.js:6]: Nope, I don't believe so (at least not without some real hackery).

Comment (sergiitk) [inline test/helpers/test-helper.js:6]: Thought so. Thank you.

**Pull request merged**